src/_data/side-nav.yml

```diff
@@ -43,6 +37,14 @@
           permalink: /guides/language/effective-dart/usage
         - title: Design
           permalink: /guides/language/effective-dart/design
+    - title: Extension methods
```

Comment:
I think this very specific topic should move below the much more general spec, tour, and type system topics

Reply:
The problem is that after the list reaches a certain length, it's hard to find items, so I tend to go to alpha order so it's predictable. The tour is the most visited page by far in this list, but I suspect that most visitors don't get to it from this sidenav.

Reply:
Sure, but we don't need extension methods to be easily locatable in the ToC. I don't think it's something we need users to stumble into organically. If they need it, they will find in in the tour, or search for it.

On the other hand we do want users to stumble into the tour and type system pages.